en/docs/develop/customizations/customizing-the-developer-portal/advanced-customization.md

```diff
@@ -1,26 +1,26 @@
 # Advanced Customization
 
-### prerequisites
+### Prerequisites
 
 #### NodeJS and NPM
 
-NodeJS is the platform needed for the ReactJS development. 
+NodeJS is a platform required for ReactJS development. 
 
-The devportal user interface can be customized simply without touching the React codebase or CSS for most common cases. But for advanced use cases, it's required to modify the react code base.
+The user interface of the Developer Portal can be customized simply without editing the React codebase or  the CSS in most cases. You will be required to modify the react code base, if you need to do advanced customizations.
 
-1. Navigate to <API-M_HOME>/repository/deployment/server/jaggeryapps/devportal/  in a terminal and run the following command.
+1. Navigate to `<API-M_HOME>/repository/deployment/server/jaggeryapps/devportal/`  in a terminal and run the following command.
 
 ```js
 npm ci
 ```
-2. Run the following to start the npm build. Note that it will continuously watch for any changes and rebuild the project.  
+2. Run the command given below, to start the npm build. Note that it will continuously watch for any changes and rebuild the project.  
 ```
 npm run build:dev
 ```
-3. If you are planning to completely rewrite the UI, then it's OK to start making changes for devportal/source/. But if you want to override a certain React Component / File from source/src/ folder, you need to do it in devportal/override/src folder. You do not have to copy the entire directory, only copy the desired file/files.
+3. If you are required to rewrite the UI completely, you can make changes in the `devportal/source` folder.
+If you want to override a specific React Component or a File from the `source/src/` folder, you need to do it in the `devportal/override/src` folder by only copying the desired file/files.
```

Comment:
Better to leave a blank line to push this content into the next line and indent it properly under 3.